data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/sink/opensearch/index/IndexConfiguration.java

```diff
@@ -55,6 +57,7 @@ public class IndexConfiguration {
     private final String documentIdField;
     private final String routingField;
     private final long bulkSize;
+    private final long flushTimeout;
```

Comment:
If we could use a Duration to be consistent between the other data prepper durations that would be ideal.

Reply:
Discussed offline - requires a non-trivial refactor since the OpenSearch sink setting logic doesn't use the common parser that can consume DataPrepperDurationSerializer. This is something we should look into for a followup